src/mingwX64Main/kotlin/io/ktor/generator/cli/CliGenerator.kt

```diff
@@ -0,0 +1,24 @@
+package io.ktor.generator.cli
+
+import io.ktor.client.*
+import io.ktor.generator.cli.installer.*
+import io.ktor.generator.cli.utils.*
+import okio.FileSystem
+import okio.Path.Companion.toPath
+
+class WindowsKtorInstaller(client: HttpClient, ktorBackendHost: String) : KtorInstaller(client, ktorBackendHost) {
```

Comment:
I'm not sure about inheritance here, it's not really needed
I would make it as class instances and all logic move to `expect/actual`.